### PR TITLE
V0.4.0

### PR DESCRIPTION
**Pull request by johndgiese**

### CODE REVIEW

File: docs/source/index.rst [hunk 1]
```diff
@@ -70,24 +70,28 @@ Details
 Change Log
 ==========
 
-Version 0.1.5
------------
+Version 0.4.0
+-------------
+- Ignore DICOMDIR files
```
Comment (johndgiese) [inline docs/source/index.rst:75]: I think we may need to add a comment or two about the HighBit change.

Comment (dillonwilliams) [inline docs/source/index.rst:75]:
```suggestion
- Ignore DICOMDIR files
- Allow HighBit and BitsStored attributes to be non-uniform when combining slices
- Support combining channeled slices
- Drop support for Python 3.4
```

Comment (johndgiese) [reply docs/source/index.rst:75]: I added a couple of other items as well in a direct commit.

**Comment by johndgiese** [comment]
@dillonwilliams let me know how this last commit looks; if its good I'll go ahead and push v0.4 to pypi

**Comment by johndgiese** [comment]
Thank you for the review!

**Pull request merged**